cmds/clients/contestcli-http/main.go

```diff
@@ -149,12 +183,40 @@ func run(verb string) error {
 		encoder.SetIndent("", " ")
 		err := encoder.Encode(apiErr)
 		if err != nil {
-			return fmt.Errorf("cannot re-encode httplistener.HTTPAPIResponse object: %v", err)
+			return "", fmt.Errorf("cannot re-encode httplistener.HTTPAPIResponse object: %v", err)
 		}
 		indentedJSON = buffer.Bytes()
 	}
 	// this is the only thing we want on stdout - the JSON-formatted response,
 	// so it can be piped to other tools if desired.
-	fmt.Println(string(indentedJSON))
-	return nil
+	return string(indentedJSON), nil
 }
+ 
+func wait(params url.Values) (string, error) {
+	
+	var finished bool
+	time.Sleep(jobWaitPoll)
+	
+	resp, err := request("status", params)
+	if err != nil {
+		return "", err
+	}
+	
+	var parsedResp map[string]interface{}
+	if err := json.Unmarshal([]byte(resp), &parsedResp); err != nil {
+		return "", fmt.Errorf("cannot decode json respnse: %v", err)
```

Comment:
In this project we have agreed to use `%w` instead of `%v` for error objects, since after Go 1.13 this allows wrapping errors without losing the original types.

Also typo: "respnse" -> "response"

Reply:
I'm leaving %v since this is what is used for the rest of the errors in the file. This can be refactored for all of them in another PR maybe.

Reply:
OK, I won't insist. Our agreement was to avoid refactor PRs of this type, and just slowly migrate things, but this is acceptable